extensions/firestore-sheets-sync/extension.yaml

```
@@ -0,0 +1,119 @@
+name: sync-firestore-sheets
```

```suggestion
name: sync-firestore-sheets
```

Would suggest changing the name to match the convention of the other extensions, e.g. the service name goes first, so `firestore-sync-google-sheets`